### ISSUE TITLE
[v1.2-Elements-Input.Text.InlineAction.json]: Label 'Text input with an inline action' is not accessible by voice over using swipe gesture.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7361]
https://microsoft.visualstudio.com/OS/_workitems/edit/37853116
